File: day_3_Dimik OJ (12).cpp
#include<bits/stdc++.h>
using namespace std;

int main() {

	//freopen("input.txt", "r", stdin);
	//freopen("output.txt", "w", stdout);

	int cases, n, result, num;
	cin >> cases;
	while (cases--)
	{
		cin >> num;

		result = 0;
		n = 5;
		while (num / n >= 1) {
			result += (int) num / n;
			n *= 5;
		}
		cout << result << endl;
	}


	return 0;
}

File: day_3_Dimik OJ (12).py
case = int(input())
while case:
    num = int(input())
    result = 0 
    power = 1
    while True:
        base = 5**power
        result += num//base
        divide = num/base
        if(divide<1):
            break
        power+=1 
    print(result)
    case-=1

